tunneler/tunneler.py

```diff
@@ -43,7 +43,7 @@ def identify_tunnel(self, server, remote_port):
         Return list of tunnel names.
         """
         names = []
-        for (name, tunnel) in self.config.tunnels.iteritems():
+        for (name, tunnel) in self.config.tunnels.items():
```

Comment:
Python 3 does not include `iteritems()` - `items()` now returns an iterable. This does mean that in Python 2 a copy of the dict will be made in memory. I decided that config is never going to be large enough for this to be a significant problem.